index.html

```diff
@@ -1,33 +1,30 @@
-<!DOCTYPE HTML>
-
-<html>
-	<head>
-		<title>Minimo Blog Challenge</title>
-		<meta charset="utf-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<link rel="stylesheet" href="css/normalize.css" />
-		<link href="https://fonts.googleapis.com/css?family=Playfair+Display|Ubuntu" rel="stylesheet">
-		<link rel="stylesheet" href="css/styles.css" />
-		<script src="js/picturefill.min.js" async></script>
-	</head>
-	<body>
-		<header>
-				<img class="minimo" src="images/Minimo.jpg" alt="Minimo logo">
-			<div class="main-nav">
-				<ul>
-					<li><a href="#lifestyle">lifestyle</a></li>
-					<li><a href="#photodiary">photodiary</a></li>
-					<li><a href="#music">music</a></li>
-					<li><a href="#travel">travel</a></li>
-				</ul>
-			</div>
-			<div>
-				<picture>
-					<source srcset="images/Header_photo.jpg 1059w, images/Header_photo_rectangle.jpg 778w, images/Header_photo_square.jpg 429w" sizes="90vw" />
-					<img src="images/Header_photo.jpg" alt="picture of two women surrounded by sand" />
-				</picture>
-			</div>
-		</header>
-	</body>
-</html>
-
+<!DOCTYPE HTML>
+
+<html>
+	<head>
+		<title>Minimo Blog Challenge</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="css/normalize.css" />
+		<link href="https://fonts.googleapis.com/css?family=Playfair+Display|Ubuntu" rel="stylesheet">
+		<link rel="stylesheet" href="css/styles.css" />
+	</head>
+	<body>
+		<div class="container">
+			<header class="clearfix">
+					<img class="minimo" src="images/Minimo.jpg" alt="Minimo logo">
+				<div class="main-nav">
+					<ul>
+					  <li><a href="#lifestyle">lifestyle</a></li>
+						<li><a href="#photodiary">photodiary</a></li>
+						<li><a href="#music">music</a></li>
+						<li><a href="#travel">travel</a></li>
+					</ul>
+				</div>
+			</header>
+			<div class="header-photo">
+			</div>

```

Comment:
Wszystko zależy od potrzeb. W tym przypadku ja nie widzę żadnych przeciwskazań by był to najzwyklejszy `img`. Przy okazji polecam Ci gdzieś na górze (za normalizem) sobie ustawić `img { max-width: 100%; }` - zazwyczaj się przydaje :). I masz ogarnięta responsywność obrazków. Teraz ten obrazek wygląda na mobile słabo - jest cholernie wysoki (500px), ale przez proporcje nic prawie na nim nie widać. Da się to ogarnąć sztuczką z ustawieniem paddingu zamiast wysokości ale moim zdaniem najzwyklejszy `img` szczególnie na tym etapie tutaj wystarczy :)

Tu masz proste demo pokazujące o co chodzi z tym trikiem z paddingiem. https://jsfiddle.net/norin89/ynnohmjb/ - ogólnie chodzi o to, że wysokość to X% z szerokości czyli proporcje grafiki są cały czas zachowane. W taki sposób wstawia się też np. wideo Youtuba na strony by się skalowało ładnie :)